processing/data_collection/gazette/spiders/base.py

```diff
@@ -9,14 +9,19 @@
 
 
 class BaseGazetteSpider(scrapy.Spider):
-    def __init__(self, start_date=None, *args, **kwargs):
+    def __init__(self, start_date=None, end_date=None, *args, **kwargs):
         super(BaseGazetteSpider, self).__init__(*args, **kwargs)
 
         if start_date is not None:
-            parsed_data = dateparser.parse(start_date)
+            parsed_data = dateparser.parse(start_date, settings={"DATE_ORDER": "DMY"})
             if parsed_data is not None:
                 self.start_date = parsed_data.date()
 
+        if end_date is not None:
```

Comment:
Considering how we will run the spiders in production, I don't think we would need an `end_date`. Mainly we will get only the gazettes from the last day.

Reply:
Ok. But that provides ways to rerun dates that failed with `RetryError` or something like that. Don't you think that would be desirable?